pontoon/teams/views.py

```diff
@@ -251,6 +251,7 @@ def ajax_permissions(request, locale):
             "project_locales": project_locales,
             "hide_project_selector": hide_project_selector,
             "community_builder_badge": community_builder_level,
+            "username": request.user,
```

Comment:
The `request` object is passed to the templates via `pontoon.base.context_processor`, so no need to pass it again here.